server/routes/api/v1/unit/create.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (express) {
 
     // Example of data in JSON format
     // unit.add({availability_qty: 3, trigger_qty:4, replenish_qty:5, description: "pink", weight_lbs: 4}
-    unit.create({availability_qty: clientUnitPost.availability_qty, trigger_qty: clientUnitPost.trigger_qty, replenish_qty: clientUnitPost.replenish_qty, description: clientUnitPost.description, weight_lbs: clientUnitPost.weight_lbs},
+    unit.add({availability_qty: clientUnitPost.availability_qty, trigger_qty: clientUnitPost.trigger_qty, replenish_qty: clientUnitPost.replenish_qty, description: clientUnitPost.description, weight_lbs: clientUnitPost.weight_lbs},
```

Comment:
according to the models its add i did pull master and then changed them according to what the models say i can change it back though